lib/cloudant.js

```diff
@@ -114,5 +114,28 @@ Cloudant.prototype.getModelObjectSettings = function(mo) {
   return undefined;
 };
 
+Cloudant.prototype.ping = function(cb) {
+  debug('Cloudant.prototype.ping');
+  var self = this;
+  var driverInst = self.getDriverInst();
+  if (driverInst) {
+    driverInst.db.list(function(err, result) {
+      returnCB(err, result);
+    });
```

Comment:
```
    driverInst.db.list(function(err, result) {
       returnCB(err, result);
     });
```
can be changed to:
```
     driverInst.db.list(returnCB(err, result));
```
@kjdelisle @jannyHou to me It is much cleaner and neater...